scripts/node.sh

```diff
@@ -736,6 +746,19 @@ if ! ${multi_key}; then
    elif [ ! -f "${blspass}" ]; then
       err 10 "can't find the ${blspass} file"
    fi
+else
+   for f in ${blsfolder}/*.key 
+   do
+      if [ ! -f $f ]; then
+         err 10 "could not find bls key file: $f"
+      fi
+      passfile=${blsfolder}/$(basename "${f%.*}").pass
+      if [ ! -f "$passfile" ]; then
+         unset -v passphrase
+         read -rsp "Enter passphrase for the BLS key file $f: " passphrase
+         echo ${passphrase} | tee $passfile
```

Comment:
this is very risky to save the passphrase to a local file w/o user's awareness.  You need to WARN user about this.  Ask user to confirm. Also, think about removing the file after the node is started.

Reply:
@LeoHChen added notification for user. Is this sufficient? To implement automatic delete of these files, I need to do that inside harmony program, which is not good. Any other way to do it as part of node.sh?

Reply:
add a parallel background process to delete the file after started harmony program.

Reply:
done.